target/linux/ramips/dts/kng_rc.dts

```diff
@@ -0,0 +1,183 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
```

Comment:
Please include <[dt-bindings/input/input.h](http://lxr.free-electrons.com/source/include/dt-bindings/input/linux-event-codes.h?v=4.4)> and  <[dt-bindings/gpio/gpio.h](http://lxr.free-electrons.com/source/include/dt-bindings/gpio/gpio.h?v=4.4)> here as well.

Use the constants GPIO_ACTIVE_LOW and GPIO_ACTIVE_HIGH afterwards in stead of 1 and 0 in the gpio parameters.

Use the Key constants instead of the hex values in linux,code.